docs/userguide.md

```diff
@@ -0,0 +1,91 @@
+*	[First run](#part_first_run)
+*	[Adding a station](#part_add_station)
+*	[Start/Stop streaming a station](#part_start)
+*	[Managing stations](#part_manage)
+	*	[The station panel](#manage_panel)
+	*	[Adding a station using Shoutcast URL](#manage_add)
+	*	[Probing a station](#manage_check)
+	*	[Deleting a station](#manage_delete)
+
+## Installation <a name = "part_install"></a>
+StreamRadio is available on HaikuDepot on both 32-bit and 64-bit Haiku. To install it, open HaikuDepot, and search for the term *StreamRadio*. Select *install* to download the package to your computer.
+
+You can also build StreamRadio from source to get a newer (but unstable) version of 
+StreamRadio. The source is available [here](https://github.com/HaikuArchives/Haiku-Radio).
+
+## First run <a name = "part_first_run"></a>
+
+![FirstRun](screenshots/01_First_run_GUI.png)
+
+When you launch StreamRadio for the first time, you will see an empty window, as you haven't added any radio stations yet. To begin, click on *Search* on the menu bar.
+
+## Add new radio stations <a name = "part_add_station"></a>
+
+![AddNewStation](screenshots/02_Add_New_Stations_1.png)
+
+To add new radio stations, select *Search* on the menu bar to begin. A dialog box will appear, allowing you to choose radio stations to add to your collection.
+
+#### Service
+Here, you can choose between two services:
+
+- [Radio Station](http://www.radio-browser.info) - A free community radio station browser.
+- [listenlive.eu](http://www.listenlive.eu/) - An experimental server, currently down.
+Adding your own radio internet radio service is currently not supported.
+
+Click on the ![Web](bnWeb.png) icon next to the service name to open the homepage of that service.
+
+#### Search by
+This menu allows you to select your search criteria. Depending on the radio service you choose, you can search based on keywords, genre, and countries.
+
+#### Search for
+This can either be a menu or a text box, according to the option you chose for *Search by*. Type in your keyword, or select your preferred type here.
+
+After you've done with your choices, press ![SearchIcon](bnSearch.png) or hit <kbd>Enter</kbd> to start your search. The search may take up to a minute.
+
+![NewStation](screenshots/03_Add_New_Stations_2.png)
+
+The results will appear in the box below. Select your preferred station, then press *Add* to add it to your list of stations. You can continue searching and adding stations like this.
+
+When you're finished, close *Find Stations* to return to the main window.
+
+## Start/Stop streaming a station <a name = "part_start"></a>
+To start streaming a station, simply click on the little square on the bottom left corner of the station's icon. The status bar should then indicate the name of the station you're currently playing.  
+![Play](screenshots/04_Play_a_station.png)  
+Click on the corresponding square on another station to switch to that station. To stop streaming, click on the square again.  
+
```

Comment:
There is no square when the stream is running, maybe rephrase?
![SR-running](https://user-images.githubusercontent.com/16057090/72207399-19759b80-3499-11ea-889f-a2f7a2b4ecd5.png)

Reply:
There is in my PC, both the version from HaikuDepot and the version compiled from source.

Reply:
![image](https://user-images.githubusercontent.com/57174311/72207429-b2bda580-34cb-11ea-8e8e-4de4c01698fd.png)
Is this an error?
I've never seen the play button before in StreamRadio.

Reply:
That's probably before the stream got started

Reply:
Once the stream started you should get the play button ...

Reply:
No, the stream has started for a while, I'm sure of that.
Anyway, it may be related to my system, so I've adjusted the guide to match with normal behavior.